iot-hub/Samples/service/DeviceStreamingSample/ServiceClientStreamingSample.csproj

```diff
@@ -15,7 +15,7 @@
   </ItemGroup>
 
   <ItemGroup>
-  	<PackageReference Include="Microsoft.Azure.Devices" Version="1.27.0-preview-004" />
+  	<PackageReference Include="Microsoft.Azure.Devices" Version="1.27.2" />
```

Comment:
Device Streaming Sample is a preview only feature. Updating the NuGet package to a GA package might break the sample.